armi/reactor/components/component.py

```diff
@@ -178,6 +178,13 @@ class Component(composites.Composite, metaclass=ComponentType):
         component types within the ARMI source code. All primitive shapes (such as a
```

Comment:
I can't comment high enough but there are a few spots here where Component should be capitalized (imo) and another instance of "provides the interface for" - let's replace that with something like "This class serves as a base class which all Component types within ARMI are built upon."